nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2393,19 +2395,53 @@ When the DFM makes changes to the dataflow, the node that receives the request t
+
+Flowfiles that remain on a disconnected node can be rebalanced to other active nodes in the cluster via offloading.  In the Cluster Management dialog, select the "Offload" icon (image:iconOffload.png["Offload Icon"]) for a Disconnected node. This will stop all processors, terminate all processors, stop transmitting on all remote process groups and rebalance flowfiles to the other connected nodes in the cluster.
+
+image::offloading-node-cluster-mgt.png["Offloading Node in Cluster Management UI"]
+
+Nodes that remain in "Offloading" state due to errors encountered (out of memory, no network connection, etc.) can be reconnected to the cluster by restarting NiFi on the node. Offloaded nodes can be either reconnected to the cluster (by selecting Connect or restarting NiFi on the node) or deleted from the cluster.
+
+image::offloaded-node-cluster-mgt.png["Offloaded Node in Cluster Management UI"]
+
+==== Delete Nodes
+
+There are cases where a DFM may wish to continue making changes to the flow, even though a node is not connected to the cluster. In this case, the DFM may elect to delete the node from the cluster entirely. In the Cluster Management dialog, select the "Delete" icon (image:iconDelete.png["Delete Icon"]) for a Disconnected or Offloaded node. Once deleted, the node cannot be rejoined to the cluster until it has been restarted.
+
+==== Decommission Nodes
+
+The steps to decommission a node and remove it from a cluster are as follows:
+
+1. Disconnect the node.
+2. Once disconnect completes, offload the node.
+3. Once offload completes, delete the node.
+4. Once the delete request has finished, stop/remove the NiFi service on the host.
+
+==== NiFi Toolkit Node Commands
 
-A DFM may manually disconnect a node from the cluster. But if a node becomes disconnected for any other reason (such as due to lack of heartbeat),
-the Cluster Coordinator will show a bulletin on the User Interface. The DFM will not be able to make any changes to the dataflow until the issue
-of the disconnected node is resolved. The DFM or the Administrator will need to troubleshoot the issue with the node and resolve it before any
-new changes may be made to the dataflow. However, it is worth noting that just because a node is disconnected does not mean that it is not working;
-this may happen for a few reasons, including that the node is unable to communicate with the Cluster Coordinator due to network problems.
+As an alternative to the UI, the following NiFi Toolkit CLI commands can be used for retrieving a single node, retrieving a list of nodes, and connecting/disconnecting/offloading/deleting nodes:
 
-There are cases where a DFM may wish to continue making changes to the flow, even though a node is not connected to the cluster.
-In this case, they DFM may elect to remove the node from the cluster entirely through the Cluster Management dialog. Once removed,
-the node cannot be rejoined to the cluster until it has been restarted.
+* `nifi get-node`
```

Comment:
Should these commands be more verbose?  Adding examples including parameters for using the TLS Toolkit CLI commands securely would be beneficial.  Also, talking about the response types would be useful, since the simple response type is more of a human readable summary, and the json response is the complete entity that's returned from the API endpoint.

Reply:
I see where you are coming from.  As I noted in the related Jira (NIFI-5659), was planning on adding more in depth coverage of these specific commands and other CLI commands in a separate NiFI Toolkit Guide.  Currently, no examples exist in the Admin Guide for the CLI commands and was trying to be consistent.  Examples do exist in the ReadMe (https://github.com/apache/nifi/blob/master/nifi-toolkit/nifi-toolkit-cli/README.md) where the simple and json output types are also covered.